### ISSUE TITLE
Missing texture and working screen - Web Displays and Flan's Convolution as well as other mods

### ISSUE THREAD

**Issue by iamsamaritan** [montoyo/webdisplays#208]
See TLDR at bottom for quick read

I've done some testing to verfiy the compatiblity with Flan's and Web Displays. My findings are as follows

Minecraft Forge 1.12.2 (2860) - latest*

Web Displays 1.1 (Listed as remastered on curseforge as I have to use curseforge to distribute packs) - current*
MCEF 1.11b (same as above) - current*

Here's whats working:
Flan's Mod (5.10.0) - current*
Simple Parts Content Pack (5.7.2) - current*
Modern Weapons Content Pack (5.8.2) - current*
WW2 Content Pack (5.7.2) - current*
Ye Olde Content Pack (5.8.2) - current*
Zombie Content Pack (5.8.5) -current*

Here's what I found causing the missing texture issue at least for my packs:
Flan's Mod: Apocalypse Content Pack (5.10.0) - current*

Testing another pack (issue #112) Wings of Freedom is the conflict

Other content packs untested

These are also tested to be working
[Bad Wither No Cookie - Reloaded (by Kreezxil)](https://www.curseforge.com/minecraft/mc-mods/bad-wither-no-cookie-reloaded)
[RandomTweaks (by TheRandomLabs)](https://www.curseforge.com/minecraft/mc-mods/randomtweaks)
[Ye Olde Pack for Flan's Mod (by FlansGames)](https://www.curseforge.com/minecraft/mc-mods/ye-olde-pack-for-flans-mod)
[Davincis Vessels (by darkevilmac)](https://www.curseforge.com/minecraft/mc-mods/davincis-vessels)
[Better Diving (by meldexun)](https://www.curseforge.com/minecraft/mc-mods/better-diving)
[Torch Slabs Mod (by EndlesNights)](https://www.curseforge.com/minecraft/mc-mods/torchslabs-mod)
[BiblioCraft (by Nuchaz)](https://www.curseforge.com/minecraft/mc-mods/bibliocraft)
[Dynamic Surroundings (by OreCruncher)](https://www.curseforge.com/minecraft/mc-mods/dynamic-surroundings)
[Clumps (by Jaredlll08)](https://www.curseforge.com/minecraft/mc-mods/clumps)
[Armory Expansion (by AshuraNoYami)](https://www.curseforge.com/minecraft/mc-mods/armory-expansion)
[Not Enough Wands (by romelo333)](https://www.curseforge.com/minecraft/mc-mods/not-enough-wands)
[Fermion Lib (by TheSilkMiner)](https://www.curseforge.com/minecraft/mc-mods/fermion-core)
[AE2 Wireless Terminal Library (by TheRealp455w0rd)](https://www.curseforge.com/minecraft/mc-mods/ae2wtlib)
[AppleCore (by squeek502)](https://www.curseforge.com/minecraft/mc-mods/applecore)
[Ender IO (by crazypants_mc_the_second)](https://www.curseforge.com/minecraft/mc-mods/ender-io)
[Nature's Compass (by Chaosyr)](https://www.curseforge.com/minecraft/mc-mods/natures-compass)
[Foam​Fix (by asiekierka)](https://www.curseforge.com/minecraft/mc-mods/foamfix-optimization-mod)
[BlockUI (by H3lay)](https://www.curseforge.com/minecraft/mc-mods/blockui)
[Compact Machines (by davenonymous)](https://www.curseforge.com/minecraft/mc-mods/compact-machines)
[Comforts (Forge) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/comforts)
[Botania (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/botania)
[Just Enough Items (JEI) (by mezz)](https://www.curseforge.com/minecraft/mc-mods/jei)
[Immersive Railroading (by cam72cam)](https://www.curseforge.com/minecraft/mc-mods/immersive-railroading)
[ExtraPlanets (by MJRLegends)](https://www.curseforge.com/minecraft/mc-mods/extraplanets)
[Macaw's Bridges (by sketch_macaw)](https://www.curseforge.com/minecraft/mc-mods/macaws-bridges)
[Crash To Main Menu (by WildBamaBoy)](https://www.curseforge.com/minecraft/mc-mods/crash-to-main-menu)
[BdLib (by bdew)](https://www.curseforge.com/minecraft/mc-mods/bdlib)
[Inventory Tweaks [1.12 only] (by JimeoWan)](https://www.curseforge.com/minecraft/mc-mods/inventory-tweaks)
[Matter Overdrive: Legacy Edition (by Buuz135)](https://www.curseforge.com/minecraft/mc-mods/matteroverdrive-legacy)
[The Aether (by GildedGames)](https://www.curseforge.com/minecraft/mc-mods/aether)
[Integration Foregoing (by Jackyy)](https://www.curseforge.com/minecraft/mc-mods/integration-foregoing)
[Open Modular Passive Defense (by Keridos)](https://www.curseforge.com/minecraft/mc-mods/ompd)
[Custom fences and walls (by leuchtraptor)](https://www.curseforge.com/minecraft/mc-mods/customfence)
[Cyclops Core (by kroeser)](https://www.curseforge.com/minecraft/mc-mods/cyclops-core)
[Carry On (by Tschipp)](https://www.curseforge.com/minecraft/mc-mods/carry-on)
[Chisel (by tterrag1098)](https://www.curseforge.com/minecraft/mc-mods/chisel)
[Brandon's Core (by brandon3055)](https://www.curseforge.com/minecraft/mc-mods/brandons-core)
[Macaw's Doors (by sketch_macaw)](https://www.curseforge.com/minecraft/mc-mods/macaws-doors)
[Ender Storage 1.8.+ (by covers1624)](https://www.curseforge.com/minecraft/mc-mods/ender-storage-1-8)
[JustEnoughIDs (by Runemoro)](https://www.curseforge.com/minecraft/mc-mods/jeid)
[More Overlays (by feldim245)](https://www.curseforge.com/minecraft/mc-mods/more-overlays)
[Thermal Innovation (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/thermal-innovation)
[Mantle (by mDiyo)](https://www.curseforge.com/minecraft/mc-mods/mantle)
[Tinkers' Addons (by oitsjustjose)](https://www.curseforge.com/minecraft/mc-mods/tinkers-addons)
[ProjectEX (by FTB)](https://www.curseforge.com/minecraft/mc-mods/projectex)
[Pixelmon (by PixelmonMod)](https://www.curseforge.com/minecraft/mc-mods/pixelmon)
[FTB Utilities (Forge) (by FTB)](https://www.curseforge.com/minecraft/mc-mods/ftb-utilities-forge)
[Torchmaster (by xalcon)](https://www.curseforge.com/minecraft/mc-mods/torchmaster)
[Guide-API (by TehNut)](https://www.curseforge.com/minecraft/mc-mods/guide-api)
[MekanismOres (by phantamanta44)](https://www.curseforge.com/minecraft/mc-mods/mekanismores)
[Fence Overhaul (by TheWeatherPony)](https://www.curseforge.com/minecraft/mc-mods/fence-overhaul)
[ICBM - Classic (by QueenOfMissiles)](https://www.curseforge.com/minecraft/mc-mods/icbm-classic)
[Toast Control (by Shadows_of_Fire)](https://www.curseforge.com/minecraft/mc-mods/toast-control)
[GunpowderLib (by Jackyy)](https://www.curseforge.com/minecraft/mc-mods/gunpowderlib)
[Unloader (by Unnoen)](https://www.curseforge.com/minecraft/mc-mods/unloader)
[Integrated Dynamics (by kroeser)](https://www.curseforge.com/minecraft/mc-mods/integrated-dynamics)
[Simply Backpacks (by Flanks255)](https://www.curseforge.com/minecraft/mc-mods/simply-backpacks)
[Open Modular Turrets (by Keridos)](https://www.curseforge.com/minecraft/mc-mods/openmodularturrets)
[EnderCore (by tterrag1098)](https://www.curseforge.com/minecraft/mc-mods/endercore)
[Baubles (by Azanor13)](https://www.curseforge.com/minecraft/mc-mods/baubles)
[OMLib (by Keridos)](https://www.curseforge.com/minecraft/mc-mods/omlib)
[Simple Parts Pack for Flan's Mod (by FlansGames)](https://www.curseforge.com/minecraft/mc-mods/simple-parts-pack-for-flans-mod)
[OreLib (by OreCruncher)](https://www.curseforge.com/minecraft/mc-mods/orelib)
[LootBags (by Mina_the_Engineer)](https://www.curseforge.com/minecraft/mc-mods/lootbags)
[Open Glider (by gr8pefish)](https://www.curseforge.com/minecraft/mc-mods/open-glider)
[SecretRoomsMod (by AbrarSyed)](https://www.curseforge.com/minecraft/mc-mods/secretroomsmod)
[OpenBlocks (by OpenMods)](https://www.curseforge.com/minecraft/mc-mods/openblocks)
[CraftTweaker (by Jaredlll08)](https://www.curseforge.com/minecraft/mc-mods/crafttweaker)
[Redstone Flux (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/redstone-flux)
[Draconic Evolution (by brandon3055)](https://www.curseforge.com/minecraft/mc-mods/draconic-evolution)
[Pam's HarvestCraft (by pamharvestcraft)](https://www.curseforge.com/minecraft/mc-mods/pams-harvestcraft)
[MrCrayfish's Furniture Mod (by MrCrayfish)](https://www.curseforge.com/minecraft/mc-mods/mrcrayfish-furniture-mod)
[Morph (by ohaiiChun)](https://www.curseforge.com/minecraft/mc-mods/morph)
[Thaumic JEI (by Buuz135)](https://www.curseforge.com/minecraft/mc-mods/thaumic-jei)
[ConnectedTexturesMod (by tterrag1098)](https://www.curseforge.com/minecraft/mc-mods/ctm)
[FTB Ultimine (Forge) (by FTB)](https://www.curseforge.com/minecraft/mc-mods/ftb-ultimine-forge)
[AE2 Wireless Interface Terminal (by TheRealp455w0rd)](https://www.curseforge.com/minecraft/mc-mods/wireless-interface-terminal)
[Multi-Piston (by H3lay)](https://www.curseforge.com/minecraft/mc-mods/multi-piston)
[More Chickens (by GenDeathrow)](https://www.curseforge.com/minecraft/mc-mods/more-chickens)
[Pixeldex (by qwbarch)](https://www.curseforge.com/minecraft/mc-mods/pixeldex)
[Universal Mod Core (by cam72cam)](https://www.curseforge.com/minecraft/mc-mods/universal-mod-core)
[Applied Energistics 2 (by AlgorithmX2)](https://www.curseforge.com/minecraft/mc-mods/applied-energistics-2)
[Bewitchment (Legacy) (by sunconure11)](https://www.curseforge.com/minecraft/mc-mods/bewitchment-legacy)
[Custom Loading Screen (by AlexIIL)](https://www.curseforge.com/minecraft/mc-mods/better-loading-screen)
[Additional Pipes for Buildcraft (by MultipleMonomials)](https://www.curseforge.com/minecraft/mc-mods/additional-pipes-for-buildcraft)
[Mouse Tweaks (by YaLTeR)](https://www.curseforge.com/minecraft/mc-mods/mouse-tweaks)
[Macaw's Furniture (by sketch_macaw)](https://www.curseforge.com/minecraft/mc-mods/macaws-furniture)
[Farming for Blockheads (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/farming-for-blockheads)
[Mystical Agradditions (by BlakeBr0)](https://www.curseforge.com/minecraft/mc-mods/mystical-agradditions)
[FTB Library (Forge) (Legacy) (by FTB)](https://www.curseforge.com/minecraft/mc-mods/ftb-library-legacy-forge)
[Corail Tombstone (by Corail_31)](https://www.curseforge.com/minecraft/mc-mods/corail-tombstone)
[NoNausea (by Lumien231)](https://www.curseforge.com/minecraft/mc-mods/nonausea)
[Storage Drawers (by Texelsaur)](https://www.curseforge.com/minecraft/mc-mods/storage-drawers)
[spark (by Iucko)](https://www.curseforge.com/minecraft/mc-mods/spark)
[Cooking for Blockheads (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/cooking-for-blockheads)
[Ex Compressum (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/ex-compressum)
[Security Craft (by Geforce132)](https://www.curseforge.com/minecraft/mc-mods/security-craft)
[Immersive Posts (by TwistedGate)](https://www.curseforge.com/minecraft/mc-mods/immersiveposts)
[RFTools (by McJty)](https://www.curseforge.com/minecraft/mc-mods/rftools)
[Thermal Expansion (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/thermal-expansion)
[BetterFps (by Guichaguri)](https://www.curseforge.com/minecraft/mc-mods/betterfps)
[Roguelike Dungeons (by Greymerk)](https://www.curseforge.com/minecraft/mc-mods/roguelike-dungeons)
[RFTools Dimensions (by McJty)](https://www.curseforge.com/minecraft/mc-mods/rftools-dimensions)
[Dark Utilities (by DarkhaxDev)](https://www.curseforge.com/minecraft/mc-mods/dark-utilities)
[McJtyLib (by McJty)](https://www.curseforge.com/minecraft/mc-mods/mcjtylib)
[Galacticraft Legacy (by TeamGalacticraft)](https://www.curseforge.com/minecraft/mc-mods/galacticraft-legacy)
[Item Filters (by LatvianModder)](https://www.curseforge.com/minecraft/mc-mods/item-filters)
[Thermal Dynamics (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/thermal-dynamics)
[Better Builder's Wands (by Portablejim)](https://www.curseforge.com/minecraft/mc-mods/better-builders-wands)
[Extra Utilities (by Forge_User_06804776)](https://www.curseforge.com/minecraft/mc-mods/extra-utilities)
[AutoRegLib (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/autoreglib)
[Expanded Equivalence (by Zeitheron)](https://www.curseforge.com/minecraft/mc-mods/expanded-equivalence)
[Mod Name Tooltip (by mezz)](https://www.curseforge.com/minecraft/mc-mods/mod-name-tooltip)
[Even More TNT (by rolik1997)](https://www.curseforge.com/minecraft/mc-mods/even-more-tnt)
[Fast Leaf Decay (by olafskiii)](https://www.curseforge.com/minecraft/mc-mods/fast-leaf-decay)
[Extreme Reactors (by ZeroNoRyouki)](https://www.curseforge.com/minecraft/mc-mods/extreme-reactors)
[Custom Main Menu (by Lumien231)](https://www.curseforge.com/minecraft/mc-mods/custom-main-menu)
[Patchouli (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/patchouli)
[MixinBooter (by Rongmario)](https://www.curseforge.com/minecraft/mc-mods/mixin-booter)
[Domum Ornamentum (by OrionOnline)](https://www.curseforge.com/minecraft/mc-mods/domum-ornamentum)
[Prefab (by wuestman)](https://www.curseforge.com/minecraft/mc-mods/prefab)
[Quark (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/quark)
[AppleSkin (by squeek502)](https://www.curseforge.com/minecraft/mc-mods/appleskin)
[CoFH World (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/cofh-world)
[Integrated Tunnels (by kroeser)](https://www.curseforge.com/minecraft/mc-mods/integrated-tunnels)
[More Planets - Galacticraft Add-on (by SteveKunG)](https://www.curseforge.com/minecraft/mc-mods/more-planets-gc-addon)
[No Mob Spawning on Trees (by oldjunyi)](https://www.curseforge.com/minecraft/mc-mods/no-mob-spawning-on-trees)
[Morpheus (by Quetzi)](https://www.curseforge.com/minecraft/mc-mods/morpheus)
[CodeChicken Lib 1.8.+ (by covers1624)](https://www.curseforge.com/minecraft/mc-mods/codechicken-lib-1-8)
[Immersive Petroleum (by Flaxbeard)](https://www.curseforge.com/minecraft/mc-mods/immersive-petroleum)
[FTB Quests (Forge) (by FTB)](https://www.curseforge.com/minecraft/mc-mods/ftb-quests-forge)
[Immersive Engineering (by BluSunrize)](https://www.curseforge.com/minecraft/mc-mods/immersive-engineering)
[iChunUtil (by ohaiiChun)](https://www.curseforge.com/minecraft/mc-mods/ichunutil)
[RandomPatches (Forge) (by TheRandomLabs)](https://www.curseforge.com/minecraft/mc-mods/randompatches-forge)
[Mekanism Tools (by bradyaidanc)](https://www.curseforge.com/minecraft/mc-mods/mekanism-tools)
[Tinkers' Tool Leveling (by bonusboni)](https://www.curseforge.com/minecraft/mc-mods/tinkers-tool-leveling)
[Blood Magic (by WayofTime)](https://www.curseforge.com/minecraft/mc-mods/blood-magic)
[No Recipe Book (by Seneschal_Luwin)](https://www.curseforge.com/minecraft/mc-mods/no-recipe-book)
[Akashic Tome (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/akashic-tome)
[AmbientSounds 5 (by CreativeMD)](https://www.curseforge.com/minecraft/mc-mods/ambientsounds)
[Astral Sorcery (by HellFirePvP)](https://www.curseforge.com/minecraft/mc-mods/astral-sorcery)
[OpenBlocks Elevator (by vsngarcia)](https://www.curseforge.com/minecraft/mc-mods/openblocks-elevator)
[Thermal Tinkering (by samboy063)](https://www.curseforge.com/minecraft/mc-mods/thermal-tinkering)
[Draconic Additions (by FoxMcloud5655)](https://www.curseforge.com/minecraft/mc-mods/draconicadditions)
[Disenchanter (The Disenchanter Mod) (by impelon)](https://www.curseforge.com/minecraft/mc-mods/the-disenchanter-mod)
[OpenModsLib (by OpenMods)](https://www.curseforge.com/minecraft/mc-mods/openmodslib)
[Bookshelf (by DarkhaxDev)](https://www.curseforge.com/minecraft/mc-mods/bookshelf)
[Wireless Crafting Terminal (by TheRealp455w0rd)](https://www.curseforge.com/minecraft/mc-mods/wireless-crafting-terminal)
[FastFurnace (by Shadows_of_Fire)](https://www.curseforge.com/minecraft/mc-mods/fastfurnace)
[stimmedcow : NoMoreRecipeConflict (by Forge_User_08823382)](https://www.curseforge.com/minecraft/mc-mods/stimmedcow-nomorerecipeconflict)
[SpatialCompat (by MrDj200)](https://www.curseforge.com/minecraft/mc-mods/spatialcompat)
[OpenComputers (by Sangar_)](https://www.curseforge.com/minecraft/mc-mods/opencomputers)
[Just Enough Resources (JER) (by way2muchnoise)](https://www.curseforge.com/minecraft/mc-mods/just-enough-resources-jer)
[Hammer Lib (by Zeitheron)](https://www.curseforge.com/minecraft/mc-mods/hammer-lib)
[Cyclic (by Lothrazar)](https://www.curseforge.com/minecraft/mc-mods/cyclic)
[Industrial Foregoing (by Buuz135)](https://www.curseforge.com/minecraft/mc-mods/industrial-foregoing)
[Flan's Mod (by FlansGames)](https://www.curseforge.com/minecraft/mc-mods/flans-mod-5-5-2)
[Project Intelligence (by brandon3055)](https://www.curseforge.com/minecraft/mc-mods/project-intelligence)
[ProjectE Integration (by TagnumElite)](https://www.curseforge.com/minecraft/mc-mods/projecte-integration)
[VanillaFix (by Runemoro)](https://www.curseforge.com/minecraft/mc-mods/vanillafix)
[MovingWorld (by darkevilmac)](https://www.curseforge.com/minecraft/mc-mods/movingworld)
[Chicken Chunks 1.8.+ (by covers1624)](https://www.curseforge.com/minecraft/mc-mods/chicken-chunks-1-8)
[Random Things (by Lumien231)](https://www.curseforge.com/minecraft/mc-mods/random-things)
[ReAuth (by TechnicianLP)](https://www.curseforge.com/minecraft/mc-mods/reauth)
[Modern Weapons Pack for Flan's Mod (by FlansGames)](https://www.curseforge.com/minecraft/mc-mods/modern-weapons-pack-for-flans-mod)
[KleeSlabs (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/kleeslabs)
[FTB Backups (Forge) (by FTB)](https://www.curseforge.com/minecraft/mc-mods/ftb-backups-forge)
[Construct's Armory (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/constructs-armory)
[Moar Tinkers (by AshuraNoYami)](https://www.curseforge.com/minecraft/mc-mods/moar-tinkers)
[Chameleon (by Texelsaur)](https://www.curseforge.com/minecraft/mc-mods/chameleon)
[FTB Money (Forge) (by FTB)](https://www.curseforge.com/minecraft/mc-mods/ftb-money-forge)
[RE-Sound (by Zeitheron)](https://www.curseforge.com/minecraft/mc-mods/resound)
[MJRLegends Lib (by MJRLegends)](https://www.curseforge.com/minecraft/mc-mods/mjrlegendslib)
[Mekanism (by bradyaidanc)](https://www.curseforge.com/minecraft/mc-mods/mekanism)
[Ex Nihilo: Creatio (by BloodWorkXGaming)](https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-creatio)
[Macaw's Trapdoors (by sketch_macaw)](https://www.curseforge.com/minecraft/mc-mods/macaws-trapdoors)
[Integrated Terminals (by kroeser)](https://www.curseforge.com/minecraft/mc-mods/integrated-terminals)
[WW2 Pack for Flan's Mod (by FlansGames)](https://www.curseforge.com/minecraft/mc-mods/ww2-pack-for-flans-mod)
[Macaw's Roofs (by sketch_macaw)](https://www.curseforge.com/minecraft/mc-mods/macaws-roofs)
[Equivalent Integrations (by pkmnfrk)](https://www.curseforge.com/minecraft/mc-mods/equivalent-integrations)
[The One Probe (by McJty)](https://www.curseforge.com/minecraft/mc-mods/the-one-probe)
[Simply Light (by Flanks255)](https://www.curseforge.com/minecraft/mc-mods/simply-light)
[Mike's Mods Library (by pkmnfrk)](https://www.curseforge.com/minecraft/mc-mods/mikes-mods-library)
[Structurize (by H3lay)](https://www.curseforge.com/minecraft/mc-mods/structurize)
[Biomes O' Plenty (by Forstride)](https://www.curseforge.com/minecraft/mc-mods/biomes-o-plenty)
[Resource Loader (by Lumien231)](https://www.curseforge.com/minecraft/mc-mods/resource-loader)
[CoFH Core (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/cofh-core)
[Inventory Pets (by Purplicious_Cow_)](https://www.curseforge.com/minecraft/mc-mods/inventory-pets)
[ProjectE (by SinKillerJ)](https://www.curseforge.com/minecraft/mc-mods/projecte)
[Waystones (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/waystones)
[Enchantment Descriptions (by DarkhaxDev)](https://www.curseforge.com/minecraft/mc-mods/enchantment-descriptions)
[Track API (by cam72cam)](https://www.curseforge.com/minecraft/mc-mods/track-api)
[Industrial Craft (by sfPlayer1)](https://www.curseforge.com/minecraft/mc-mods/industrial-craft)
[xXx_MoreToolMats_xXx (a PlusTiC fork without the evil, and apparently some new bugs idk) (by TeamDman)](https://www.curseforge.com/minecraft/mc-mods/plusticminusbad)
[Mekanism Generators (by bradyaidanc)](https://www.curseforge.com/minecraft/mc-mods/mekanism-generators)
[CreativeCore (by CreativeMD)](https://www.curseforge.com/minecraft/mc-mods/creativecore)
[JourneyMap (by techbrew)](https://www.curseforge.com/minecraft/mc-mods/journeymap)
[End Portal Recipe (Forge) (by Serilum)](https://www.curseforge.com/minecraft/mc-mods/end-portal-recipe)
[Faster Ladder Climbing (by maddachshund)](https://www.curseforge.com/minecraft/mc-mods/faster-ladder-climbing)
[Controlling (by Jaredlll08)](https://www.curseforge.com/minecraft/mc-mods/controlling)
[Macaw's Fences and Walls (by sketch_macaw)](https://www.curseforge.com/minecraft/mc-mods/macaws-fences-and-walls)
[SwingThroughGrass (by exidex)](https://www.curseforge.com/minecraft/mc-mods/swingthroughgrass)
[FastWorkbench (by Shadows_of_Fire)](https://www.curseforge.com/minecraft/mc-mods/fastworkbench)
[Tinkers' Complement (by KnightMiner)](https://www.curseforge.com/minecraft/mc-mods/tinkers-complement)
[Sampler (by sfPlayer1)](https://www.curseforge.com/minecraft/mc-mods/sampler)
[Thermal Foundation (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/thermal-foundation)
[Industrial Wires (by malte0811)](https://www.curseforge.com/minecraft/mc-mods/industrial-wires)
[TAIGA (Tinkers alloying addon) (by zkafaceTV)](https://www.curseforge.com/minecraft/mc-mods/taiga-tinkers-alloying-addon)
[Crafting Tweaks (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/crafting-tweaks)
[p455w0rd's Library (by TheRealp455w0rd)](https://www.curseforge.com/minecraft/mc-mods/p455w0rds-library)
[AI Improvements (by QueenOfMissiles)](https://www.curseforge.com/minecraft/mc-mods/ai-improvements)
[Buildcraft (by Covert_Jaguar)](https://www.curseforge.com/minecraft/mc-mods/buildcraft)
[Placebo (by Shadows_of_Fire)](https://www.curseforge.com/minecraft/mc-mods/placebo)
[Tiny Progressions (by Kashdeya)](https://www.curseforge.com/minecraft/mc-mods/tiny-progressions)
[Mob Grinding Utils (by vadis365)](https://www.curseforge.com/minecraft/mc-mods/mob-grinding-utils)
[Spice of Life: Carrot Edition (by lordcazsius)](https://www.curseforge.com/minecraft/mc-mods/spice-of-life-carrot-edition)
[Mystical Agriculture (by BlakeBr0)](https://www.curseforge.com/minecraft/mc-mods/mystical-agriculture)
[CraftingNetherStar (by ZeckG)](https://www.curseforge.com/minecraft/mc-mods/craftingnetherstar)
[SteveKunG's Lib (by SteveKunG)](https://www.curseforge.com/minecraft/mc-mods/stevekungs-lib)
[Collective (Forge) (by Serilum)](https://www.curseforge.com/minecraft/mc-mods/collective)
[Integrated Crafting (by kroeser)](https://www.curseforge.com/minecraft/mc-mods/integrated-crafting)
[MalisisDoors (by Ordinastie)](https://www.curseforge.com/minecraft/mc-mods/malisisdoors)
[AlchemyLib (by Dark_Arcana)](https://www.curseforge.com/minecraft/mc-mods/alchemylib)
[BuildCraft RF: ReFluxified (by JackRaidenPH)](https://www.curseforge.com/minecraft/mc-mods/buildcraft-rf)
[It's the little things (by Zlepper)](https://www.curseforge.com/minecraft/mc-mods/its-the-little-things)
[MalisisBlocks (by Ordinastie)](https://www.curseforge.com/minecraft/mc-mods/malisisblocks)
[MalisisCore (by Ordinastie)](https://www.curseforge.com/minecraft/mc-mods/malisiscore)
[Ding (by ohaiiChun)](https://www.curseforge.com/minecraft/mc-mods/ding)
[Thaumcraft (by Azanor13)](https://www.curseforge.com/minecraft/mc-mods/thaumcraft)
[Macaw's Windows (by sketch_macaw)](https://www.curseforge.com/minecraft/mc-mods/macaws-windows)
[AE2 Stuff (by bdew)](https://www.curseforge.com/minecraft/mc-mods/ae2-stuff)
[Tesla Core Lib (by face_of_cat)](https://www.curseforge.com/minecraft/mc-mods/tesla-core-lib)
[ComputerCraft (by DanTwoHundred)](https://www.curseforge.com/minecraft/mc-mods/computercraft)
[MalisisSwitches (by Ordinastie)](https://www.curseforge.com/minecraft/mc-mods/malisisswitches)
[Actually Additions (by Ellpeck)](https://www.curseforge.com/minecraft/mc-mods/actually-additions)
[Shadowfacts' Forgelin (by ShadowfactsDev)](https://www.curseforge.com/minecraft/mc-mods/shadowfacts-forgelin)
[MineColonies (by H3lay)](https://www.curseforge.com/minecraft/mc-mods/minecolonies)
[Zombie Content Pack for Flan's Mod (by FlansGames)](https://www.curseforge.com/minecraft/mc-mods/zombie-content-pack-for-flans-mod)
[Iron Chests (by ProgWML6)](https://www.curseforge.com/minecraft/mc-mods/iron-chests)
[ZeroCore 2 (by ZeroNoRyouki)](https://www.curseforge.com/minecraft/mc-mods/zerocore)
[Thermal Cultivation (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/thermal-cultivation)
[Alchemistry (by Dark_Arcana)](https://www.curseforge.com/minecraft/mc-mods/alchemistry)
[CraftableCreativeModifier (by DaveyL2013)](https://www.curseforge.com/minecraft/mc-mods/craftablecreativemodifier)
[Tinkers Construct (by mDiyo)](https://www.curseforge.com/minecraft/mc-mods/tinkers-construct)
[Tick Dynamic (by The_Fireplace)](https://www.curseforge.com/minecraft/mc-mods/tick-dynamic)
[The Twilight Forest (by Benimatic)](https://www.curseforge.com/minecraft/mc-mods/the-twilight-forest)
[Cucumber Library (by BlakeBr0)](https://www.curseforge.com/minecraft/mc-mods/cucumber)
[Common Capabilities (by kroeser)](https://www.curseforge.com/minecraft/mc-mods/common-capabilities)
[Chickens (by setycz)](https://www.curseforge.com/minecraft/mc-mods/chickens)
[Flux Networks (by sonar_sonic)](https://www.curseforge.com/minecraft/mc-mods/flux-networks)
[Better Foliage (by octarine_noise)](https://www.curseforge.com/minecraft/mc-mods/better-foliage)

TLDR
Flan's Mod: Apocalypse Content Pack is at least one source of the missing texture issue

Upon further testing based on issue #112 Wings of Freedom causes the same issue

*current as of 12/12/2022 at 8 PM Eastern US

**Comment by ChaosAgent202** [comment]
Essentials is also on a list of incompatibilities due to MCEF.
